trade-finance-manager-api/src/v1/controllers/deal.controller.update-tfm-underwriter-managers-decision.test.js

```diff
@@ -0,0 +1,173 @@
+const api = require('../api');
+const { updateTfmUnderwriterManagersDecision } = require('./deal.controller');
+const mapTfmDealStageToPortalStatus = require('../mappings/map-tfm-deal-stage-to-portal-status');
+
+const MOCK_DEAL_BSS_MIA = require('../__mocks__/mock-deal-MIA-submitted');
+const MOCK_DEAL_GEF_MIA = require('../__mocks__/mock-gef-deal-MIA');
+const MOCK_NOTIFY_EMAIL_RESPONSE = require('../__mocks__/mock-notify-email-response');
+const CONSTANTS = require('../../constants');
+const { mockUpdateDeal } = require('../__mocks__/common-api-mocks');
+
+jest.mock('../api');
+
+const sendEmailApiSpy = jest.fn(() => Promise.resolve(MOCK_NOTIFY_EMAIL_RESPONSE));
+
+const updatePortalBssDealStatusSpy = jest.fn(() => Promise.resolve({}));
+const updatePortalGefDealStatusSpy = jest.fn(() => Promise.resolve({}));
+const addUnderwriterCommentToGefDealSpy = jest.fn(() => Promise.resolve({}));
+const addPortalDealCommentSpy = jest.fn(() => Promise.resolve({}));
+const findBankByIdSpy = jest.fn(() => Promise.resolve({ emails: [] }));
+const findOneTeamSpy = jest.fn(() => Promise.resolve({ email: [] }));
+
+describe('update tfm underwriter managers decision', () => {
+  const bssDealId = MOCK_DEAL_BSS_MIA._id;
+  const comments = 'Testing';
+  const internalComments = '';
+  const userFullName = 'Test User';
+
+  beforeEach(() => {
+    sendEmailApiSpy.mockClear();
+    api.sendEmail = sendEmailApiSpy;
+
+    sendEmailApiSpy.mockClear();
+    api.sendEmail = sendEmailApiSpy;
+    updatePortalBssDealStatusSpy.mockClear();
+    api.updatePortalBssDealStatus = updatePortalBssDealStatusSpy;
+    api.updatePortalGefDealStatus = updatePortalGefDealStatusSpy;
+    api.addUnderwriterCommentToGefDeal = addUnderwriterCommentToGefDealSpy;
+    api.addPortalDealComment = addPortalDealCommentSpy;
+    findBankByIdSpy.mockClear();
+    api.findBankById = findBankByIdSpy;
+
+    findOneTeamSpy.mockClear();
+    api.findOneTeam = findOneTeamSpy;
+
+    api.updateDeal.mockReset();
+    mockUpdateDeal();
+  });
+
+  afterAll(() => {
```

Comment:
As you mentioned -- probably don't need this

Reply:
Removed, thanks!